app/src/main/res/font/karla.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<font-family xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+    <font
+        android:font="@font/karla_regular"
```

Comment:
Hiya, I checked that the attribute for `font` is only used for API level 26 or higher. Our target sdk is at API level 23 which is behind the required 26. While this wouldn't really hurt our app that much, can you find any alternatives? Other than that, it's an OK from me. Do tell me if you found or couldn't found any other alternatives

Reply:
I would suggest yours to remove the conflicting element tags that only works for android API 26 or newer, and try these on emulator/devices.. these tags even removed from these XML may work properly on newer Android API too.
---
Mungkin coba dihapus beberapa element tag yg hanya bisa terpakai di Android API 26/terbaru.. barangkali tanpa itu sebetulnya tetap jalan dengan aman di device/emulator bahkan di android terbaru sekalipun.

Coba dicomment/reply/chat aja klo ternyata ada masalah/bug gara2 tag tersebut dihapus

Reply:
I think that it will be better to move the `android:font` to the v26 layout resource if possible though. Wdyt?